### PR TITLE
project addition: added JS project

### PR DESCRIPTION
**Pull request by WajeehaCodes**
## Related Issue

PR #884

### CODE REVIEW

File: src/DB/projects.json [hunk 1]
```diff
@@ -1704,5 +1704,23 @@
...
+        "Instagram": "",
+        "YouTube": "",
+        "Twitter": ""
+      },
+      "Projects": [
+        {
+          "link": "https://github.com/WajeehaCodes/Shopping-Cart-JS",
+          "title": "Shopping Cart JS",
+          "description": "The JavaScript Shopping Cart project is a web application that allows users to browse and purchase products. It provides a seamless and interactive shopping experience by implementing a shopping cart functionality using JavaScript.",
+          "tech": ["HTML", "CSS", "JavaScript"]
```
Comment (priyankarpal) [inline src/DB/projects.json:1722]: change to `Javascript`
```suggestion
          "tech": ["HTML", "CSS", "Javascript"]
```

**Comment by priyankarpal** [comment]
please add the issue number to the PR description

**Comment by WajeehaCodes** [comment]
I have added it. Can you please guide me that if I am writing it in the wrong way.
Screenshot 2023-05-27 at 11 52 39 AM

**Comment by priyankarpal** [comment]
> I have added it. Can you please guide me that if I am writing it in the wrong way. Screenshot 2023-05-27 at 11 52 39 AM

it's the PR number add your issue number. if you haven't created an issue please create & add the issue number https://github.com/priyankarpal/ProjectsHut/issues/new?assignees=&labels=%F0%9F%8C%B1+chore%3A+projects+addition%2C%F0%9F%91%8D++status%3A+approved%2Cgssoc23%2Clevel1&projects=&template=project-add.yml&title=data%3A+project+addition+by+

**Pull request closed without merging**